### DISCUSSION TITLE
Can I use the repo to convert HTML table to PNG image?

### DISCUSSION THREAD

**Discussion by zydjohnHotmail** [mono/SkiaSharp#1961] [Q&A]
I have a simple HTML table, like this:

<p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(36, 41, 47); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Hello:<br style="box-sizing: border-box;"></p>
<table style="width:100%">
  <tr>
    <th>#</th>
    <th>Goal</th>
    <th>Time</th>
    <th>Score</th>
  </tr>
  <tr>
    <td style="border: 3px solid red; ">1</td>
    <td style="border: 3px solid red; ">Otavio</td>
    <td style="border: 3px solid red; ">24:10</td>
    <td style="border: 3px solid red; ">1-0</td>
  </tr>
  <tr>
    <td>2</td>
    <td>L. Diaz</td>
    <td>36:50</td>
    <td>2-0</td>
  </tr>
  <tr>
    <td>3</td>
    <td>M. Taremi</td>
    <td>77:20</td>
    <td>3-0</td>
  </tr>
  <tr>
    <td>4</td>
    <td>Riccieli</td>
    <td>90:51</td>
    <td>3-1</td>
  </tr>
</table>

Can I use the repo to convert the HTML table to PNG file?</span>

**Comment by mattleibow** [comment]
Not directly unfortunately. SkiaSharp is more of a lower level engine.

However, you may be able to take advantage of some markdown package that uses SkiaSharp - if that package supports HTML in markdown. Or there may be another library that does this and uses SkiaSharp as a backend.

@toptensoftware, does RichTextKit support HTML?
